### DISCUSSION TITLE
Install a visual regression testing tool

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#127] [Ideas]
This would be useful to sign off on UI diffs.

**Comment by kentcdodds** [comment]
Agreed.

Trouble is I've not used a tool like this in years, and the landscape has changed a lot. I'll need to evaluate the options.

If anyone has suggestions, please don't hesitate to provide them!

**Reply by vinstah** [reply]
I've used https://github.com/garris/BackstopJS which was helpful against a live site and also chromatic for testing against storybook builds

**Reply by vinstah** [reply]
also since there is playwrite integrtion already https://playwright.dev/docs/test-snapshots
